rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/OracleTableMetaCache.java

```diff
@@ -76,9 +76,17 @@ private TableMeta resultSetMetaToSchema(DatabaseMetaData dbmd, String tableName)
         String[] schemaTable = tableName.split("\\.");
         String schemaName = schemaTable.length > 1 ? schemaTable[0] : dbmd.getUserName();
         tableName = schemaTable.length > 1 ? schemaTable[1] : tableName;
+        if (schemaName != null) {
```

Comment:
schemaName can be null?

Reply:
I check it later, maybe is the unit test problem.